proxy-libs/helioslambda/lambda.go

```diff
@@ -13,20 +13,47 @@ import (
 
 var InstrumentedSymbols = [...]string{"Start", "StartWithContext", "StartWithOptions"}
 
-type httpHeaders struct {
+type apiGatewayEvent struct {
 	Headers map[string]string `json:"headers"`
 }
 
+type eventBridgeEvent struct {
+	Detail      map[string]string `json:"detail"`
+	TraceHeader string            `json:"trace-header"`
+}
+
 func heliosEventToCarrier(eventJSON []byte) propagation.TextMapCarrier {
-	var headers httpHeaders
+	const traceParentHeader = "Traceparent"
```

Comment:
we can move them outside from the. method no? or you don't want to change the original file too much?

Reply:
For now they're used locally only, so I'd prefer leaving it this way